### PR TITLE
Document for Connecting a service to an application (SBO related content on ODC)

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: enterprise-`4.9`
- **JIRA issues**: [RHDEVDOCS-3303](https://issues.redhat.com/browse/RHDEVDOCS-3303)
- **Preview pages**: [Connecting a service to an application](https://deploy-preview-36702--osdocs.netlify.app/openshift-enterprise/latest/applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.html), [Viewing application composition using the Topology view](https://deploy-preview-36702--osdocs.netlify.app/openshift-enterprise/latest/applications/odc-viewing-application-composition-using-topology-view.html)
- **SME Review**: @pedjak , @baijum 
- **QE review**: Approved @pmacik 
- **Peer-review**:  @Preeticp

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@pedjak @baijum @pmacik Please review and provide your feedback. Thanks!

File: applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+== Creating a visual connection between components
+
+You can depict an intent to connect application components using the visual connector.
+
+This procedure walks through an example of creating a visual connection between a PostgreSQL Database service and a Spring PetClinic sample application.
+
+.Prerequisites
+
+* Ensure that you have created and deployed a Spring PetClinic sample application using the *Developer* perspective.
```
Comment (pedjak) [inline applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:21]: is there details in the doc how to do this step?

File: applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* Ensure that you have created and deployed a Crunchy PostgreSQL database instance using the *Developer* perspective.
```
Comment (pedjak) [inline applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:22]: is there details in the doc how to do this step?

File: applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+. Click on the `spring-petclinic-rest` Deployment to see the *Overview* panel. Under the *Details* tab, click the edit icon in the *Annotations* section to see the *Key = `app.openshift.io/connects-to`* and *Value = `[{"apiVersion":"apps/v1","kind":"Deployment","name":"hippo-pgbouncer"}]`* annotation added to the Deployment.
+
+Similarly you can create other applications and components and establish visual connections between them.
+
+
+== Creating a binding connection between components
+
+[NOTE]
+====
+Currently, a few specific Operators like the *PostgresSQL Database* Operator's service instances are bindable.
```
Comment (pedjak) [inline applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:37]: there are multiple PostgresSQL operator available, which one do you mean exactly?

Comment (Srivaralakshmi) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:37]: @pedjak @pmacik We need those inputs from you. Kindly provide.

File: applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:22]: Will provide links to Getting started with service binding topics under additional resources section.

File: applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:21]: Will provide links to Getting started with service binding topics under the additional resources section

File: applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc (hunk 3 above)
Comment (pedjak) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:37]: @pmacik please correct me, but I think that currently there is no operator that can be used properly from ODC?

Comment (pmacik) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:37]: @pedjak The Crunchy Postgresql as shown and used in quick-start can be used in ODC.

Comment (pmacik) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:37]: @Srivaralakshmi correct me if I'm wrong, but I believe that the reference in text to `PostgreSQL Database Operator` means "that operator that we installed in Prerequisities", it's not the actual name of the operator.

Comment (pmacik) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:37]: As far as I know the only PostgreSQL DB operators supported by SBO at GA are [Crunchy PostgreSQL Operator](https://operatorhub.io/operator/postgresql) and [Cloud Native PostgreSQL](https://operatorhub.io/operator/cloud-native-postgresql).
@pedjak please confirm

Comment (pmacik) [reply applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc:37]: However, I'm not sure if tha Cloud Native one is supported by ODC itself

**Review by Preeticp — COMMENTED** [review]
@Srivaralakshmi Looks good overall, ave left a few suggestions PTAL. Also please add the 2 screenshots as discussed. It would add clarity to the procedure.
I would also suggest adding a sentence to the prerequisite in both the procedures, something on these lines:  
* Ensure that you have created and deployed a Crunchy PostgreSQL database instance using the *Developer* perspective. This instance has the following three components...
So that it is clear to the end users.

**Pull request merged**